src/extensions/default/bramble/lib/iframe-browser.js

```diff
@@ -180,46 +147,29 @@ define(function (require, exports, module) {
     /**
      * Used to show second pane, change lilveDevButton background and close the detached preview
      */
-    function attachPreview() {
```

Comment:
@humphd did we stop supporting this feature? I can't seem to find this being called anywhere

Reply:
Do I have to clarify something here?

Reply:
Yeah, we are not using this feature any more.